lib/linguist/languages.yml

```diff
@@ -3309,6 +3309,17 @@ ObjDump:
   tm_scope: objdump.x86asm
   ace_mode: assembly_x86
   language_id: 256
+ObjectScript:
+  type: programming
+  extensions:
+  - ".cls"
+  - ".inc"
+  - ".int"
+  - ".mac"
```

Comment:
From a very quick look using GitHub's very limited search capabilities, I couldn't find a single `.mac` extensioned file outside of your org that matched the heuristic suggesting this extension is not popular enough for inclusion right now.

Please remove this extension or provide a more specific search result that illustrates this extension meets our "hundreds of repositories" requirement.